spec/parser_spec.lua

```diff
@@ -555,6 +568,17 @@ describe("Parser /", function()
             ]], "this expression in a statement position is not a function call")
         end)
 
+        it("disallows too many arguments", function ()
+           local args = ''
+           for i = 1, 200 do
+              args = args .. i .. ', '
+           end
+           args = args ..'201'
+            assert_statements_error(string.format([[
+                f(%s)
+            ]], args), "too many arguments (limit is 200)")
```

Comment:
The same comments for the other test case apply here (indentation, table.concat, and string.format)